src/SOS/Strike/disasm.cpp

```diff
@@ -1138,4 +1138,137 @@ LPCSTR ARM64Machine::s_SPName           = "sp";
 
 #endif // SOS_TARGET_ARM64
 
+//
+// GCEncodingInfo class member implementations
+//
 
+bool GCEncodingInfo::Initialize()
+{
+    buf = nullptr;
+    cchBufAllocation = 0;
+    cchBuf = 0;
+    curPtr = nullptr;
+    done = false;
+    hotSizeToAdd = 0;
+
+    return ReallocBuf();
+}
+
+void GCEncodingInfo::Deinitialize()
+{
+    delete[] buf;
```

Comment:
I can't remember for sure, but does delete check for null? Do you need to check buf for nullptr first?

Reply:
delete does accept nullptr (presumably it hasn't been overridden by an implementation that does not handle it)